frontend/src/features/builder/EditorContext.tsx

```diff
@@ -71,6 +71,17 @@ export const useProvideEditor = (): EditorContextProps => {
   }, [value])
 
   const saveTemplate = useCallback(async () => {
+    if (activeEditorId) {
+      return BuilderService.updateTemplate(activeEditorId, {
+        name: activeTemplateName,
+        body: [
+          {
+            type: 'TEXT',
+            data: activeEditorValue,
+          },
+        ],
+      })
+    }
```

Comment:
Consider creating a payload object since it's identical for both save and update